elrond-wasm/src/types/interaction/arg_buffer.rs

```diff
@@ -34,6 +34,16 @@ impl ArgBuffer {
 		self.arg_data.as_ptr()
 	}
 
+	/// returns a clone of the raw arg data
+	pub fn arg_data(&self) -> Vec<u8> {
```

Comment:
Better return as slice &[u8] and let the whoever uses the method to .clone() is necessary